test/roundtrip/try.txt

```diff
@@ -0,0 +1,30 @@
+;;; TOOL: run-roundtrip
```

Comment:
It would be good to add some dump tests too, to ensure that these are being decoded properly by wasm-objdump.